wrappers/winuser.go

```diff
@@ -206,14 +206,17 @@ const (
 	EDD_GET_DEVICE_INTERFACE_NAME = 0x00000001
 )
 
+type MonitorEnumProc func(hmonitor HMONITOR, hdc HDC, rect *RECT, lparam LPARAM) uintptr
```

Comment:
Let's rename this to `MONITORENUMPROC` for consistency with what's in winuser.h.  Also, the callback returns a `BOOL`, which we should be able to represent in Go as `int32` instead of `uintptr`.

Reply:
We could probably get around the issue by doing something like this:
```
fnEnumRaw := func(hmonitor uintptr, hdc uintptr, rect uintptr, lparam uintptr) uintptr {
    return uintptr(fnEnum(
        syscall.Handle(hmonitor),
        syscall.Handle(hdc),
        (*RECT)(unsafe.Pointer(rect)),
        lparam))
}
```
(I'm guessing that the only reason that it's working with the parameters being of their actual types rather than `uintptr` is that we got lucky and the types used all happened to have the same size as a pointer.  We should establish the convention of passing in a wrapper function that takes all `uintptr` so that it will work for any callbacks we add in the future that take parameters of other sizes.)

Reply:
I do not understand... Do you think that: 
`type MONITORENUMPROC func(hmonitor syscall.Handle, hdc syscall.Handle, rect *RECT, lparam uintptr) uintptr`

should be changed to:
`type MONITORENUMPROC func(hmonitor uintptr, hdc uintptr rect uintptr, lparam uintptr) uintptr`
?

Reply:
No, what I meant is to change `MONITORENUMPROC` to return `int32` to match what's in winuser.go.  Then, write a lambda taking and returning `uintptr` that wraps the function and pass the lambda to the API function.

Reply:
aaah, now I understand :) Done.